From 426c1944ba7d0eca68726d27e2d390b7055844ff Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:21 -0400
Subject: [redhat] kabi: Adding symbol __tracepoint_module_get (include/trace/events/module.h)

Message-id: <cfa260576aceaf288bfcf059ae32f22fc3fd1e2a.1410510773.git.poros@redhat.com>
Patchwork-id: 96025
O-Subject: [PATCH RHEL7.1 021/233] kabi: Adding symbol __tracepoint_module_get (include/trace/events/module.h)
Bugzilla: 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    refrenced via a DEFINE_EVENT call in module.h We expect all modules to need this.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__tracepoint_module_get
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__tracepoint_module_get
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__tracepoint_module_get

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__tracepoint_module_get b/redhat/kabi/kabi-module/kabi_ppc64/__tracepoint_module_get
new file mode 100644
index 0000000..bfc034d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__tracepoint_module_get
@@ -0,0 +1,2 @@
+#1-
+0xa3fc3d50 __tracepoint_module_get vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__tracepoint_module_get b/redhat/kabi/kabi-module/kabi_s390x/__tracepoint_module_get
new file mode 100644
index 0000000..3179099
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__tracepoint_module_get
@@ -0,0 +1,2 @@
+#1-
+0xd4df235d __tracepoint_module_get vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__tracepoint_module_get b/redhat/kabi/kabi-module/kabi_x86_64/__tracepoint_module_get
new file mode 100644
index 0000000..bfc034d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__tracepoint_module_get
@@ -0,0 +1,2 @@
+#1-
+0xa3fc3d50 __tracepoint_module_get vmlinux EXPORT_SYMBOL
-- 
1.7.1